src/garage/envs/mujoco/half_cheetah_dir_env.py

```diff
@@ -76,10 +76,10 @@ def step(self, action):
             task_name = 'backward'
         else:
             raise ValueError('task direction should be 1. or -1.')
-        infos = dict(reward_forward=forward_reward,
-                     reward_ctrl=-ctrl_cost,
-                     task_dir=self._task['direction'],
-                     task_name=task_name)
+        infos = dict(reward_forward=np.asarray([forward_reward]),
+                     reward_ctrl=np.asarray([-ctrl_cost]),
+                     task_dir=np.asarray([self._task['direction']]),
+                     task_name=np.asarray([task_name]))
```

Comment:
BTW, you can't make an `np.ndarray` of the `task_name`. That will cause things to crash later.